multimedia/gst1-plugins-base/patches/004-Synchronize-playback-support-using-alsa-start-at.patch

```diff
@@ -0,0 +1,3018 @@
+From e7da364d4cb979d9308f17f1413bd4793547e22b Mon Sep 17 00:00:00 2001
+From: Abhilash Tuse <Abhilash.Tuse@imgtec.com>
+Date: Fri, 25 Nov 2016 18:23:19 +0530
+Subject: [PATCH] Synchronize playback support using alsa start-at
+
+Change-Id: I1189aaa8e2d66f664afe89a63490b792ca928143
+Signed-off-by: Abhilash Tuse <Abhilash.Tuse@imgtec.com>
```

Comment:
A bit of a description of what the patch does would be useful here. Same for the alsa patch too.

Also, some developers put significant effort into this, perhaps mention some of the key players in signed off by.